packages/@sanity/studio-hints/src/components/serializers/CodeSnippet.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable complexity */
```

Comment:
Will this file be included by default in all studios? If so, have we measured how much refractor (and these languages) this adds to the bundle size?